doc/tutorials/gapi/table_of_content_gapi.markdown

```diff
@@ -15,3 +15,16 @@ how G-API module can be used for that.
     is ported on G-API, covering the basic intuition behind this
     transition process, and examining benefits which a graph model
     brings there.
+
+- @subpage tutorial_gapi_face_beautification
+
+    *Languages:* C++
+
+    *Compatibility:* \> OpenCV 4.1.2
```

Comment:
4.1.2 or 4.2?

Reply:
If the sample works with [OpenCV 4.1.2 release](https://github.com/opencv/opencv/tree/4.1.2) - you may keep 4.1.2.

Reply:
Thank you